File: C/1043 - Triângulo.c
#include <stdio.h>

int main(void){

    double a, b, c;

    scanf(" %lf %lf %lf", &a, &b, &c);

    if (a < b + c && b < a + c && c < a + b){

        double perimetro = a + b + c;
        printf("Perimetro = %.1f\n", perimetro);

    }else{

        double area = ((a+b) * c) / 2;
        printf("Area = %.1f\n", area);

    }

    return 0;

}

File: Python/Lista 3/1043 - Triângulo.py
temp = input()
temp = temp.split(' ')

a = float(temp[0])
b = float(temp[1])
c = float(temp[2])

isTri = ((a < b + c) and (b < c + a) and c < a + b)

if isTri:
    solution = a + b + c
    print("Perimetro = {:.1f}".format(solution))

else:
    solution = ((a+b) * c) / 2
    print("Area = {:.1f}".format(solution))
